### PR TITLE
Fix "Unknown version, custom build?" in /version

### PR DESCRIPTION
**Pull request by pisaiah**
Add new /version command that will display versions behind info instead of "Unknown version, custom build?"

From https://github.com/fungus-soft/bukkit4fabric/commit/93a74a98a42364b8eefba12dffc21cf61b9349b2

### CODE REVIEW

**Comment by hYdos** [comment]
just double checking, you are checking if it builds & runs first right?

**Comment by ramidzkh** [comment]
We already have enough gradle cursedness, I'm not going to use yet another plugin. It'd be better if it read it from the fabric.mod.json

**Comment by Ueaj-Kerman** [comment]
concerned_tater

**Comment by hYdos** [comment]
something tells me this is going to be bike shead'ed

**Comment by CoolMineman** [comment]
just hard code the version string smh

**Comment by hYdos** [comment]
honestly that would be a good solution for now

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Version file (autogenerated by gradle)
```
Comment (hYdos) [inline .gitignore]: shouldn't need comments in the gitignore

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+/src/main/java/io/github/fukkitmc/fukkit/GitVersion.java
+
```
Comment (hYdos) [inline .gitignore]: random space also shouldn't exist

**Review by hYdos — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (jearmstrong21) [reply .gitignore]: > something tells me this is going to be bike shead'ed

prediction 100

**Comment by CoolMineman** [comment]
@IsaiahPatton hydos says you have to fix the pr or he will send you an irritater

**Comment by hYdos** [comment]
i just asked to fix the pr :concern:

**Comment by hYdos** [comment]
im just going to commit stale on this. it can be reopened at a later time.

**Pull request closed without merging**